lib/errors.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+const getContent = (err, debug, translate) => {
+  var content = {};
```

Comment:
`let` makes it possible to scope to a block that isn't a function. You know what I mean.
is `var` a thing of the past?